concent_api/requirements.lock

```diff
@@ -1,12 +1,17 @@
 asn1crypto==0.24.0
 bitcoin==1.1.42
 cbor2==3.0.4
-cffi==1.11.2
+certifi==2018.1.18
+cffi==1.11.4
+chardet==3.0.4
 coincurve==6.0.0
 Django==1.11.9
 ethereum==1.6.1
+factory-boy==2.9.2
+Faker==0.8.10
```

Comment:
I don't think that `requests` depends on `factory-boy` or `Faker`. This should not be here. Please regenerate this file in a clean virtuelenv.